app/assets/javascripts/discourse/app/components/edit-topic-timer-form.gjs

```diff
@@ -80,7 +80,16 @@ export default class EditTopicTimerForm extends Component {
   }
 
   get statusType() {
-    return this.args.topicTimer.status_type;
+    const statusType = this.args.topicTimer.status_type;
+
+    if (
+      statusType === CLOSE_STATUS_TYPE &&
+      this.args.topicTimer.based_on_last_post
+    ) {
+      return CLOSE_AFTER_LAST_POST_STATUS_TYPE;
```

Comment:
This is kind of annoying to do since `CLOSED_AFTER_LAST_POST_STATUS_TYPE` is actually not a real topic timer type. It is a status type that is used when a `TopicTimer#status` is set to `close` and `TopicTimer#based_on_last_post` is `true`. The proper fix here is to introduce closed based on last post as a new timer type so we don't have to do conditionals like this but that fix will be more involved.